CONTRIBUTING.md

```diff
@@ -168,11 +168,15 @@ to `.vscode/settings.json` in your local Miri clone:
         "./cargo-miri/Cargo.toml"
     ],
     "rust-analyzer.checkOnSave.overrideCommand": [
+        "env",
+        "AUTO_OPS=42",
         "./miri",
         "check",
         "--message-format=json"
     ],
     "rust-analyzer.buildScripts.overrideCommand": [
+        "env",
+        "AUTO_OPS=42",
```

Comment:
Uh so you don't want the automatic stuff all the time then?^^
Not sure how I feel about seeing this in the docs, it was supposed to be entirely an implementation detail...

Reply:
Please at least make it `AUTO_OPS=no` so that it does not look so silly. ;) And add a comment explaining what this does.
Should we rename the env var to MIRI_AUTO_OPS to stay in our namespace+